packages/tpetra/core/test/CrsMatrix/CrsMatrix_NodeConversion.cpp

```diff
@@ -103,6 +103,7 @@ namespace {
   ////
   TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( CrsMatrix, NodeConversion, N2 )
   {
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
```

Comment:
Best practice would be to make the test not run at all, not just disable the contents of the test.  This would mean 
  1. wrapping the entire test declaration in the macro, and
  2. protecting instantiation of the test (see bottom of file) with the macro.